builders/interoperability/xcm/xc20/overview.md

```diff
@@ -7,44 +7,73 @@ description: Learn about the types of cross-chain assets on Moonbeam, in particu
 
+## Asset Reserves 
+
+### Remote Reserve Assets
+
+A remote reserve asset is a token whose canonical ledger—the source of truth for minting and burning—resides on a chain different from where it’s currently in use. In the case of xcDOT on Moonbeam, the underlying DOT tokens representing the xcDOT remain locked in Moonbeam’s sovereign account on the Polkadot relay chain, while xcDOT functions as a wrapped representation in Moonbeam’s EVM environment.
+
+Users can hold and transact with xcDOT on Moonbeam (for DeFi, governance, and more), knowing that the underlying DOT is safely locked on the relay chain. At any point, the wrapped xcDOT can be redeemed for the original DOT, effectively burning the xcDOT and unlocking the corresponding DOT tokens on Polkadot.
+
+### Local Reserve Assets 
+
+A local reserve asset on Moonbeam is a token whose canonical ledger—from an XCM perspective—resides natively on Moonbeam. In other words, Moonbeam is the asset’s home chain, where minting and burning take place. 
+
+For example, Wormhole-wrapped ETH (wETH) is considered a local reserve asset on Moonbeam, even though Ethereum is the ultimate source of ETH. Once ETH is wrapped by Wormhole and enters the Polkadot ecosystem via Moonbeam, wETH can be transferred to other parachains through [Moonbeam Routed Liquidity (MRL)](/builders/interoperability/mrl/){target=\_blank}.
+
+The important caveat is that, on a purely Ethereum-level view, ETH remains governed by and minted on Ethereum. However, from an XCM standpoint, wETH on Moonbeam is treated as a local reserve asset, meaning the canonical supply of wETH (as far as Polkadot ecosystems are concerned) exists on Moonbeam.
+
+## Multilocation Overview
+
+A [multilocation](/builders/interoperability/xcm/core-concepts/multilocations/){target=\_blank} is a structured way to pinpoint where an asset or account lives in the Polkadot/Kusama ecosystem. It specifies how many hops up or down the chain hierarchy are needed (the parents), and which detailed paths (the interior junctions) must be followed to get to a specific parachain, account, pallet, or asset.
+
+When you see an XC-20 on Moonbeam that’s not native to Moonbeam (for example, xcDOT or xcASTR), it corresponds to a foreign asset whose reserve or home is on another parachain or the relay chain.
+
+Keep in mind that multilocations are always relative to a current position. For instance, defining a multilocation on Moonbeam differs from defining one on the Polkadot relay chain, even if both refer to the same asset. The multilocation identifies the asset’s origin chain and asset ID so the XCM virtual machine knows precisely where to lock, unlock, or update tokens. It identifies that asset’s origin chain and asset ID so that XCM knows:
```

Comment:
```suggestion
Keep in mind that multilocations are always relative to a current position. For instance, defining a multilocation on Moonbeam differs from defining one on the Polkadot relay chain, even if both refer to the same asset. The multilocation identifies the asset’s origin chain and asset ID so the Cross-Consensus Virtual Machine (XCVM) knows precisely where to lock, unlock, or update tokens. It identifies that asset’s origin chain and asset ID so that XCM knows:
```

I'm not sure about "It idefnities that asset's origin chain and asset ID so that XCM knows", specifically "XCM knows" sounds kinda weird to me haha 🤷‍♀️